benches/throughput.rs

```diff
@@ -374,12 +380,139 @@ pub fn metrics(c: &mut Criterion) {
     });
 }
 
+/// Iterate through possible IP pairs restricted to 0 < ip_pair.0 < ip_pair.1 <= MAX_HBONE_WORKLOADS.
+fn next_ip_pair(ip_pair: (u8, u8)) -> (u8, u8) {
```

Comment:
Note this is only n*(n-1)/2 combos. If we wanted more new HBONE connections from the same number of IPs this could be optimized.